src/components/modals/DeleteTableModal.tsx

```diff
@@ -0,0 +1,62 @@
+import { Modal, Typography } from 'antd';
+import React, { useCallback } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useDeleteTable } from '../../hooks/tables';
+
+interface DeleteTableModalProps {
+  open: boolean;
+  closeModal: () => void;
+  tableFullName: string;
+  catalog: string;
+  schema: string;
+  table: string;
+}
+
+export function DeleteTableModal({
+  open,
+  closeModal,
+  tableFullName,
+  catalog,
+  schema,
+  table,
+}: DeleteTableModalProps) {
+  const navigate = useNavigate();
+  const mutation = useDeleteTable({
+    onSuccessCallback: () => {
+      localStorage.setItem('notification', JSON.stringify({
+        message: `${table} table successfully deleted`,
+        type: 'success'
+      }));
```

Comment:
I suggest using [notification api](https://ant.design/components/notification) with some duration for this.

Reply:
Ok, in order to handle this with the redirect, I'm going to create a `<NotificationContext />` so we can use it globally.

Reply:
+1 global context 👍